src/honeybadger.cr

```diff
@@ -0,0 +1,15 @@
+require "./honeybadger/*"
+
+module Honeybadger
+  VERSION = "0.1.0"
+
+  @@api_key = ""
+
+  def self.configure(api_key : String) : Nil
+    @@api_key = api_key
+  end
+
+  def self.api_key
+    @@api_key
+  end
```

Comment:
This file also needs to have a `Honeybadger.notify(error)` (and a section in the README documenting it—[see the one for Golang](https://docs.honeybadger.io/lib/go.html#manually-reporting-errors), for instance).

Reply:
@joshuap for sure, I'm a little lacking context about why you'd use this method manually. Should this take an exception, and use a payload builder? Should it take an instance of a payload (ie, functionally just json)?

Reply:
@robacarp it should take an exception and (probably?) use the payload builder (I'll defer to you on that). The function of this method is to manually report an error from your application code (i.e. outside of an HTTP handler). That's a very common use case in Ruby—is Crystal different?

The `Honeybadger.notify` method is also described in our [Client Library Spec](https://www.notion.so/honeybadger/Client-Library-Spec-aa891332f7874196aa0695b6d38dca66#2dbc79c2264947589e04f22dbec91490) and in our [Ruby gem](https://github.com/honeybadger-io/honeybadger-ruby/blob/2d24a36f5f560c4234f22f59153c80f32e4ce177/lib/honeybadger/agent.rb#L80).